DevelopGitHub/GitHubExtension/GitHubExtention.Preferences.WebApi/Extensions/StringExtentions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using GitHubExtention.Preferences.WebApi.Constants; 
+
+namespace GitHubExtention.Preferences.WebApi
+{
+    public static class StringExtentions
+    {
+        public static string SaveAvatarToBlobStorage(this string url,  string userName)
```

Comment:
You are created general extension which can use ANY ANY string object. Is it what you need?